api/cases/templates/caseList/index.js

```diff
@@ -0,0 +1,11 @@
+const divorce = require('./divorce.template.json');
+const sscs = require('./sscs.template.json');
+
+const templates = {
+    divorce: divorce,
+    sscs, sscs
```

Comment:
this is duplicate data ... please could we extract this out rather than having duplicate data. 

api/cases/templates/case/index.js is the exact same.

Also this has a typo and incorrectly adding sscs template twice.